src/Templates/NServiceBusDockerContainer/NServiceBusDockerContainer.csproj

```diff
@@ -3,7 +3,6 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
So if I understood correctly by moving this to the directory.build.props we still make sure our code has no warnings and we do not enforce warnings as errors on users using the template. So essentially the template is now slightly less opinionated?

Reply:
So we discussed and we think the goal of this template is not to have an opinionated stance on treat warning as errors therefore we'll remove it